### PR TITLE
Do not assume cluster name and namespace across Orchest

### PR DESCRIPTION
**Pull request by fruttasecca**
## Description

Makes it so that the controller will pass the Orchest namespace and cluster name to the relevant deployments, along with other minor adjustments.

### CODE REVIEW

**Comment by nhaghighat** [comment]
Looks good

**Review by nhaghighat — APPROVED** [review]

File: services/orchest-webserver/client/src/views/UpdateView.tsx [hunk 1]
```diff
@@ -44,19 +46,26 @@ const UpdateView: React.FC = () => {
...
+          )
             .then((json) => {
               setState((prevState) => ({
                 ...prevState,
+                clusterName: json.cluster_name,
+                namespace: json.namespace,
               }));
               console.log("Update started, polling controller.");
 
               // TODO: hardcoded namespace and cluster name values.
```
[the remark this answers is not in the document]
Comment (yannickperrenet) [reply services/orchest-webserver/client/src/views/UpdateView.tsx]: Done: https://github.com/orchest/orchest/pull/940/commits/cce4d8693c8654f14567c43ee1654c4d8057a233

**Review by yannickperrenet — APPROVED** [review]
:+1:

**Comment by yannickperrenet** [comment]
I guess this solves https://github.com/orchest/orchest/issues/866? With the caveat of collisions with third-party deployments already in the namespace, e.g. argo.

@nhaghighat @fruttasecca

**Comment by fruttasecca** [comment]
>I guess this solves https://github.com/orchest/orchest/issues/866? With the caveat of collisions with third-party deployments already in the namespace, e.g. argo.

Should be a yes at least from codebase perspective, no idea from the controller or other kind of cluster configuration POV. We should probably test it.

**Comment by yannickperrenet** [comment]
> Should be a yes at least from codebase perspective, no idea from the controller or other kind of cluster configuration POV. We should probably test it.

Awesome! Will try it out and respond on the issue.

**Pull request merged**